### PR TITLE
Clarify what model to create

### PR DESCRIPTION
**Pull request by morenoh149**
Clarify what model to create during post installation,

## General

 - [x] Make sure yo use [semantic commit messages](https://seesparkbox.com/foundry/semantic_commit_messages). 
       Examples: `"fix(google): Fixed foobar bug"`, `"feat(accounts): Added foobar feature"`.
 - [x] All Python code must be 100% pep8 and isort clean.
 - [x] Javascript code should adhere to [StandardJS](https://standardjs.com).
 - [x] If your changes are significant, please update `ChangeLog.rst`.
 - [x] Feel free to add yourself to `AUTHORS`.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -151,5 +151,5 @@ Now start your server, visit your admin pages (e.g. http://localhost:8000/admin/
 and follow these steps:
 
 1. Add a ``Site`` for your domain, matching ``settings.SITE_ID`` (``django.contrib.sites`` app).
-2. For each OAuth based provider, add a ``Social App`` (``socialaccount`` app).
+2. For each OAuth based provider, add a ``Social applications`` (inside the ``socialaccount`` app).
```
Comment (pennersr) [inline docs/installation.rst]: This should be singular (application).

Comment (morenoh149) [reply docs/installation.rst]: Just trying to match what the user sees 
Screen Shot 2019-04-25 at 6 03 57 PM

Could rephrase it to `For each OAuth based provider, add a new instance under ``Social applications``.`

Comment (pennersr) [reply docs/installation.rst]: It does not state "Add social applications" on screen. On screen, there is a link to all applications (plural), and an "Add" link, which adds a single application.
